mlir/lib/Dialect/Tosa/Utils/QuantUtils.cpp

```diff
@@ -150,10 +145,58 @@ mlir::tosa::buildConvOpQuantizationAttr(OpBuilder &builder, Value input,
       weightZp = weightPerAxisQType.getZeroPoints().front();
     }
 
-    return builder.getAttr<tosa::ConvOpQuantizationAttr>(inputZp, weightZp);
+    return std::make_pair(inputZp, weightZp);
   }
 
-  return nullptr;
+  return std::nullopt;
+}
+
+std::pair<Value, Value>
+mlir::tosa::createZPsAsConst(OpBuilder &builder, Value input, Value weight) {
```

Comment:
Other ops have `zero_points` on input/output or in the case of matmul: `a` and `b`. I think this is okay for now but just noting we may need to clean this function up later on down the line